semantic-conventions/syntax.md

```diff
@@ -168,6 +174,18 @@ The following is only valid if `type` is `event`:
   If not specified, the `prefix` is used. If `prefix` is empty (or unspecified),
   `name` is required.
 
+#### Metric semantic convention
+
+The following is only valid if `type` is `metric`:
+
+  - `metric_name`, the metric name as described by the [OpenTelemetry Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#timeseries-model). 
+  - `instrument`, the [instrument type]( https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument) 
+  that should be used to record the metric. Note that the semantic conventions must be written 
+  using the names of the synchronous instrument types (`counter`, `gauge`, `updowncounter` and `histogram`).
+  For more details: [Metrics semantic conventions - Instrument types](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#instrument-types).
+  - `unit`, the unit in which the metric is measured, which should adhere to 
+    [the guidelines](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#instrument-units). 
+
```

Comment:
You could add that all three properties are required and maybe that optional attributes can be added.

Reply:
They aren't technically required, due to what I said over here: https://github.com/open-telemetry/build-tools/pull/79#discussion_r999985536 

They are required for "concrete" metrics, but for the parent metrics, they aren't.

Reply:
see https://github.com/open-telemetry/build-tools/pull/79#discussion_r1003468865

Reply:
Closing - this was resolved in the meantime.